### PR TITLE
Added joint scale to the model objects

### PR DESCRIPTION
**Pull request by skgarvie**

### CODE REVIEW

File: public/js/sceneSubjects/Character.js [hunk 1]
```diff
@@ -22,25 +21,27 @@ function Character(characterIndex, center=false) {
...
+      const joint = new Joint(jointScale, jointColor, jointName);
 
       joints.push(joint);
 
       child.add(joint);
     });
-    if(center) {
-      rig.position.set(0,0,0);
+    if (center) {
+      rig.position.set(0, 0, 0);
```
Comment (HermanMartinus) [inline public/js/sceneSubjects/Character.js]: Nice +1

File: public/js/sceneSubjects/Character.js [hunk 2]
```diff
@@ -63,14 +64,21 @@ function Character(characterIndex, center=false) {
...
-        if (characterInfo.joints.includes(child.name) && !checked.includes(child.name)) {
+        if (characterInfo.joints.some(joint => joint.name === child.name) && !checked.includes(child.name)) {
           thingToDo(child);
           checked.push(child.name);
         }
       }
     });
   }
 
+  const resetJointColor = (joint) => {
```
Comment (HermanMartinus) [inline public/js/sceneSubjects/Character.js]: Maybe move this to Joint, so instead of calling `resetJointColor(joint)`, you just call jont.resetJointColor

**Comment by skgarvie** [comment]
Yeah. Can do same with the opacity stuff
Have no references to the material in character

**Comment by HermanMartinus** [comment]
The joint color should be set to `this.color` in initialization and reset to that when `joint.resetColor()` is called. What do you think?

**Pull request merged**